src/Hexonet/Provider.php

```diff
@@ -548,7 +472,7 @@ public function _getInfo(string $domain, $msg = 'Domain data obtained'): DomainR
         ];
         $domainInfo['locked'] = boolval(array_intersect($lockedStatuses, $domainInfo['statuses']));
 
-        if (true || !Utils::tldSupportsExplicitRenewal(Utils::getTld($domain))) {
```

Comment:
@uphlewis the `true` statement was always taking the code through this route, felt redundant, so removed it)

Reply:
Thanks - im sure that accidentally crept in after a long debugging session 🤦‍♂️